src/config.ts

```diff
@@ -22,5 +22,9 @@ export default function buildConfig(
     aqi: config.aqi ?? {},
     stats: config.stats ?? [],
     shortcuts: config.shortcuts ?? [],
+
+    // Support for card-mod styling
+    style: config.style,
+    card_mod: config.card_mod,
```

Comment:
hey, would it work if we just pass these props like this:

```suggestion
    ...config
```